plugins/woocommerce/src/StoreApi/Routes/V1/AbstractTermsRoute.php

```diff
@@ -96,6 +96,13 @@ public function get_collection_params() {
 			'default'     => true,
 		);
 
+		$params['parent'] = array(
+			'description'       => __( 'Limit result set to resources assigned to a specific parent. Applies to hierarchical taxonomies only.', 'woocommerce' ),
```

Comment:
We could use `terms` or `items` instead of `resources`. We could also be able to shorten the string in general, for example like this:


```suggestion
			'description'       => __( 'Limit results to terms with a specific parent (hierarchical taxonomies only).', 'woocommerce' ),
```